dev/tasks/run-e2e

```diff
@@ -23,8 +23,11 @@ cd ${REPO_ROOT}
 echo "Downloading envtest assets..."
 export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
 
-echo "Using direct controllers: SQLInstance,ComputeForwardingRule,GKEHubFeatureMembership"
-export KCC_USE_DIRECT_RECONCILERS=SQLInstance,ComputeForwardingRule,GKEHubFeatureMembership
+if [[ -z "${KCC_USE_DIRECT_RECONCILERS:-}" ]]; then
+  KCC_USE_DIRECT_RECONCILERS=SQLInstance,ComputeForwardingRule,GKEHubFeatureMembership
+fi
+echo "Using direct controllers: $KCC_USE_DIRECT_RECONCILERS"
+export KCC_USE_DIRECT_RECONCILERS
```

Comment:
Are these defaults? If so, shouldn't the default controller be selected via the annotation (or lack thereof) from the CRD? Or are these resources in transition?

Anyways, I think being able to expose `KCC_USE_DIRECT_RECONCILERS` to the dev tooling makes sense.